AirCasting/CreateSessionViews/ConnectingABView/ConnectingABView.swift

```diff
@@ -82,8 +78,9 @@ struct ConnectingABView: View {
         }
     }
 }
-//struct ConnectingABView_Previews: PreviewProvider {
+
+// struct ConnectingABView_Previews: PreviewProvider {
```

Comment:
we should uncomment and fix this ([The Boy Scout Rule](https://deviq.com/principles/boy-scout-rule), remember ;) I know, I know, it's annoying)

Reply:
For sure You are right, I will try to remember then (a bit annoying 😛)